.github/workflows/fg-sync-repos.yml

```diff
@@ -0,0 +1,46 @@
+name: Floodgate Repo Sync
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate a token
+        id: generate_token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ secrets.FG_SYNC_APP_ID }}
+          private-key: ${{ secrets.FG_SYNC_APP_PRIVATE_KEY }}
+          owner: ${{ github.repository_owner }}
+          repositories: "bacom-pink"
+
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+
+      - name: Clone Floodgate Repository
+        run: |
+          git clone https://github.com/adobecom/bacom-pink.git ../bacom-pink
+
+      - name: Overwrite floodgate repo files with latest from source repo
+        run: |
+          rsync -av --exclude='fstab.yaml' --exclude='.github' --exclude='.git' --exclude='.idea' ./ ../bacom-pink/
+
+      - name: Commit and Push Changes to Floodgate Repository
+        run: |
+          cd ../bacom-pink
+          git config user.email "$FG_SYNC_BOT_EMAIL"
+          git config user.name "$FG_SYNC_BOT_NAME"
+          git status          
+          git remote set-url origin https://oauth2:$GITHUB_TOKEN@github.com/adobecom/bacom-pink.git
+          git remote -v
+          git add .
+          git commit -m "Syncing bacom to bacom-pink"
```

Comment:
Couldn't we just add a remote and push to it?

Reply:
Not sure. Thoughts @sukamat ?